stylesheets/all.scss

```diff
@@ -1,103 +1,42 @@
 @import 'node_modules/lbh-frontend/lbh/all';
 @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&amp;display=swap');
 
-$screen-sm-min: 576px;
-
 * {
   font-family: Open Sans, Arial, sans-serif !important;
+  box-sizing: border-box;
 }
 
 * + * {
   margin-top: 0px;
 }
 
+html,
 body {
-  margin-top: 0;
+  padding: 0;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
+    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
 }
 
-[role='button'] {
-  cursor: pointer;
+a {
+  color: inherit;
+  text-decoration: none;
 }
 
-.govuk-tag:not([class*='govuk-tag--']) {
-  background-color: #00513f;
+[role='button'] {
+  cursor: pointer;
 }
 
 .govuk-required {
   color: #d21b1b;
 }
 
-.govuk-table__row--clickable:hover {
-  background: govuk-colour('light-grey');
-  cursor: pointer;
-}
-
-.govuk-input:disabled {
-  background: govuk-colour('light-grey');
-}
-
-.govuk--timestamp {
-  font-weight: 600;
-}
-
-.gov-weight-lighter {
-  font-weight: 100;
-}
-
-.govuk-custom-text-color {
-  color: #01674e;
-}
-
-.govuk-divider {
-  border: solid 2px #01674e;
-}
-
-.govuk-warning {
-  color: #da3711;
-}
-
-.govuk-link-button {
-  min-width: 210px;
-  padding: 10px 0;
-}
-
-.govuk-button {
-  border-radius: 4px;
-}
-
-.lbh-table-header {
-  display: flex;
-  justify-content: space-between;
-  align-items: flex-end;
-}
-
-.cancel-icon {
-  width: 20px;
-  height: 20px;
-  top: 5px;
-  position: relative;
-  cursor: pointer;
-}
-
 .sticky {
   position: -webkit-sticky;
   position: sticky;
   top: 5rem;
 }
 
-.govuk-breadcrumbs__list-item {
-  margin: 4px;
-}
-
-.govuk-fieldset__heading {
-  font-size: 20px;
-  margin: 0;
-}
-
-.govuk-heading-m {
-  margin-bottom: 40px;
-}
-
 .react-pdf__Page__canvas {
```

Comment:
I wonder if it's possible to move this and the one below into a scss component?

Could we do something like:
* put a wrapper div around the `<Document>` in PdfViewer
* In the PdfViewer css module, have something like
```
.pdf-wrapper {
 .react-pdf__Page__canvas { ... }
 .react-pdf__Page__annotations.annotationLayer {...}
}
```
which we then apply to the wrapper div?